pkg/kube/portforwarder.go

```diff
@@ -69,22 +75,25 @@ func (f *forwarder) Start() error {
 			// Build a new port forwarder.
 			fw, err := f.buildK8sPortForwarder(readyCh)
 			if err != nil {
-				errCh <- err
+				f.errCh <- fmt.Errorf("building port forwarded: %v", err)
```

Comment:
Looks like some of the logs in the original PR were deleted, why?

Reply:
the log package was not present in 1.18. Added the logs using scopes.Framework.log